src/cli/config.js

```diff
@@ -94,7 +98,16 @@ async function getConfigurationFile(filename, folder) {
         }
     }
 
-    return await getDefaultConfigurationFileTraversingParents(folder);
+    // Look at parent directories or, finally, the user's home directory
+    filepath = await getDefaultConfigurationFileTraversingParents(folder);
+    if (!filepath) {
+        const homedir_file = path.join(os.homedir(), ".svglintrc.js");
```

Comment:
For style consistency, can you use camelCase for this variable name.

Reply:
I renamed variables to use camelCase and updated the README.md file.